install.sh

```diff
@@ -297,6 +380,17 @@ install_zypper_packages( )
   done
 }
 
+install_zypper_packages_version( )
+{
+  package_dependencies=("$1")
+  package_versions=("$2")
+  for index in ${package_dependencies[*]}; do
+    printf "\033[32mInstalling \033[33m${package_dependencies[$index]} version ${package_versions[$index]} from distro package manage
+r \033[0m\n"
+    elevate_if_not_root zypper -n --no-gpg-checks install ${package_dependencies[$index]}-${package_versions[$index]}
+  done
```

Comment:
Is there a reason we do these one at a time? I know that's how the existing installs are done, but it's way faster to install everything with one call to `zypper/apt/yum/dnf`.

Reply:
Not that I know of, I created LWPMLSE-243 to keep PRs focused.